prow/config/jobs/api-1.20.yaml

```diff
@@ -0,0 +1,780 @@
+branches:
+- release-1.20
+env:
+- name: BUILD_WITH_CONTAINER
+  value: "0"
+image: gcr.io/istio-testing/build-tools:release-1.20-ff93080f0b66b1649fd6f3bfa6b63f74be377937
+jobs:
+- command:
+  - make
+  - presubmit
+  env:
+  - name: BUILD_WITH_CONTAINER
+    value: "0"
+  - name: BUILD_WITH_CONTAINER
+    value: "0"
+  image: gcr.io/istio-testing/build-tools:release-1.20-ff93080f0b66b1649fd6f3bfa6b63f74be377937
```

Comment:
So these are the duplicates that I typically get rid of. We already have this in line 4-6 above.